database/metadata/cron_triggers.yaml

```diff
@@ -1 +1,17 @@
-[]
+- name: refresh_location_crashes_view
+  webhook: http://localhost:8080/v2/query
+  schedule: '0 * * * *'
+  include_in_metadata: true
+  payload: {}
+  headers:
+    - name: x-hasura-admin-secret
+      value_from_env: CRON_TRIGGER_API_SECRET
```

Comment:
new env var `CRON_TRIGGER_API_SECRET` will need to be added to ECS configs and set to the same value as the admin secret. this value is actually ignored locally since we don't use an admin secret.